packages/content/src/lib/content-files-token.ts

```diff
@@ -1,14 +1,32 @@
-import { InjectionToken } from '@angular/core';
+import { InjectionToken, inject } from '@angular/core';
 
 import { getContentFiles } from './get-content-files';
+import { CONTENT_FILES_LIST_TOKEN } from './content-files-list-token';
 
 export const CONTENT_FILES_TOKEN = new InjectionToken<
   Record<string, () => Promise<string>>
 >('@analogjs/content Content Files', {
   providedIn: 'root',
   factory() {
     const contentFiles = getContentFiles();
+    const contentFilesList = inject(CONTENT_FILES_LIST_TOKEN);
 
-    return contentFiles;
+    const lookup: Record<string, string> = {};
+    contentFilesList.forEach((item) => {
+      lookup[item.filename] = item.slug || item.filename;
```

Comment:
So, I'm a little confused on what we should return for the content files then. I reverted this back to what's in the main branch and it can't find the content for the page any more when using the `slug` from the frontmatter.